assets/templates/login.html

```diff
@@ -1,46 +1,68 @@
-<!DOCTYPE html>
+  <!DOCTYPE html>
 <html lang="fr">
   <head>
     <meta charset="utf-8">
     <title>Cozy</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="/settings/theme.css">
-    <link rel="stylesheet" href="/assets/styles/stack.css">
+    <link rel="stylesheet" href="/assets/fonts/fonts.css">
+    <link rel="stylesheet" href="/assets/styles/login.css">
```

Comment:
Can we have a name more generic than `login.css`? I'd like to use the same CSS for the page where we ask the user to give permissions to a device (OAuth2)